docs/advanced/DAP.md

```diff
@@ -31,7 +33,45 @@ The USB serial port will appear on a Windows machine as a COM port, or on a Linu
 
 For more information on configuring your host computer to use this feature, please [see here](../getting_started/serial_communication.md).
 
+## Interface Boards
+
+Currently there are two stand alone boards commercially available that you can run DAPLink on.
+
+#### DIPDAP
+
+<span class="images">![](images/dipdap.jpg)<span>dipdap</span></span>
+
+The DIPDAP is a debug probe based on the NXP LPC11U35, supported by the DAPLink project. It provides a CMSIS-DAP interface and a serial port
+[Buy - LTEK](http://uk.rs-online.com/web/p/processor-microcontroller-development-kits/9054100/) [Buy - Minewtech](http://www.minewtech.cn/product/60653683522-804909412/DIPDAP_Board_with_matching_cable_robotic_arm.html?spm=a2700.8304367.0.0.N9LkhJ) | [Design Files](https://github.com/ARMmbed/mbed-HDK/tree/master/Production%20Design%20Projects/ARM-mbed/DAPLink/DIPDAP)
+
+#### SWDAP
+
+<span class="images">![](images/swdap.jpg)<span>swdap</span></span>
+
+The SWDAP is a debug probe based on the NXP LPC11U35, supported by the DAPLink project. It provides a CMSIS-DAP interface for programming and debugging. This design is NRND (not reccomended for new designs) because it does not have COM Port functionality, so it cannot be used for automated testing in the mbed ecosystem, which is required for validation and continuous integration testing.
+[Buy](http://uk.rs-online.com/web/p/processor-microcontroller-development-kits/9054104/) | [Design Files](https://github.com/ARMmbed/mbed-HDK/tree/master/Production%20Design%20Projects/ARM-mbed/DAPLink/SWDAP)
+
 ## Further reading
 
 * [The DAPLink GitHub repo](https://github.com/ARMmbed/DAPLink/blob/master/README.md).
 * [Debug probes built with DAPLink](https://developer.mbed.org/platforms/SWDAP-LPC11U35).
+
+## FAQ
+
+	- Q: How do I program a board using DAPLink?
+	- A: The DAPLink interface circuit will show up like a USB Drive to your computer. To program the target chip connected to the DAPLink interface chip simply dag and drop the target binary to the USB drive. DAPLink will then load the program to the target chip.
+
+	- Q: How do I program different target chips with the same DAPLink interface? (ie use a DIPDAP to program different targets)
+	- A: Currently DAPLink can only program one target. To switch the target you will need change the firmware on the DAPLink interface chip. 
```

Comment:
You can use the same firmware to program different target chips if you use a PC tool such as Keil, IAR or pyOCD. Also, by default DIPDAP should ship with firmware that disables the MSD endpoint so it can only be used to program with PC tools.